common/components/componentsBySection/FindProjects/ProjectCard.jsx

```diff
@@ -21,7 +25,7 @@ class ProjectCard extends React.PureComponent<Props> {
 
   render(): React$Node {
     return (
-      <a
+      <a style={styles}
```

Comment:
Is there a need to inject this style into the markup?

Reply:
This type of fix was the only one that actually was making sure all links at the same time were not underlining, unless there is a better method. Tried all I knew!

Reply:
Alright.  In that case can we remove the redundant lines in the .css?

Reply:
Will do!